### DISCUSSION TITLE
If internet is available, get info from stations that hear you from PSKREPORT

### DISCUSSION THREAD

**Discussion by lasantiagoa** [danleetw/FT8TW#5] [Ideas]
If the internet is available, get info from stations that hear you from PSKREPORT.

This feature will be exciting. If the internet and PSKREPORT are available, implement a feature similar to GridTracker. It queries PSKREPORTER and shows you which stations you can hear are also hearing you (especially those calling CQ).

By adding an option to the CQ Method (prioritize stations that are hearing me), we can ensure that we'll answer back to those stations where we have the biggest chance of getting a reply. 

Of course, if the internet or PSKREPORTER is unavailable, we revert to the other CQ Methods. 

73 de Luis, HB9HJU

**Comment by danleetw** [comment]
> If the internet is available, get info from stations that hear you from PSKREPORT.
> 
> This feature will be exciting. If the internet and PSKREPORT are available, implement a feature similar to GridTracker. It queries PSKREPORTER and shows you which stations you can hear are also hearing you (especially those calling CQ).
> 
> By adding an option to the CQ Method (prioritize stations that are hearing me), we can ensure that we'll answer back to those stations where we have the biggest chance of getting a reply.
> 
> Of course, if the internet or PSKREPORTER is unavailable, we revert to the other CQ Methods.
> 
> 73 de Luis, HB9HJU

"Does PSKReporter provide such an API? I only found the upload functionality."

**Reply by lasantiagoa** [reply]
Yes, Sir. Please read the Data Retrieval section of this link:

https://pskreporter.info/pskdev.html

Hope this is useful.

Luis, HB9HJU

**Reply by danleetw** [reply]
> Yes, Sir. Please read the Data Retrieval section of this link:
> 
> https://pskreporter.info/pskdev.html
> 
> Hope this is useful.
> 
> Luis, HB9HJU

I’d like to recommend a tool I often use. When my signal is received, it automatically sends an alert to my phone, and my Garmin watch vibrates. That way, I know my signal has successfully made it out.

https://hamalert.org/
